Serialization/Serialization/Serialization/ClothingArticleList.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Serialization
+{
+    [Serializable]
+    [KnownType(typeof(Shirt))]
+    [KnownType(typeof(Sock))]
+    public class ClothingArticleList<T> : List<T> where T : ClothingArticle
```

Comment:
This class should have XML commentary. A typical way to describe a class is to do something like, "Represents a list of `<see cref="ClothingArticle"/> objects.` (-0.2)